pkg/registry/core/replicationcontroller/strategy_test.go

```diff
@@ -205,3 +209,206 @@ func TestValidateUpdate(t *testing.T) {
 		t.Fatalf("expected error related to the selector")
 	}
 }
+
+// Helper function for RC tests.
+func mkValidReplicationController(tweaks ...func(rc *api.ReplicationController)) api.ReplicationController {
+	rc := api.ReplicationController{
+		ObjectMeta: metav1.ObjectMeta{Name: "abc", Namespace: metav1.NamespaceDefault},
+		Spec: api.ReplicationControllerSpec{
+			Replicas: 1,
+			Selector: map[string]string{"a": "b"},
+			Template: &api.PodTemplateSpec{
+				ObjectMeta: metav1.ObjectMeta{
+					Labels: map[string]string{"a": "b"},
+				},
+				Spec: podtest.MakePodSpec(),
+			},
+		},
+	}
+	for _, tweak := range tweaks {
+		tweak(&rc)
+	}
+	return rc
+}
+
+func TestValidateForDeclarative(t *testing.T) {
```

Comment:
@thockin We could take the organization one step farther and put these tests into a file `validation_test.go` file in the this registry directory.  The way these test are now we're appending validation tests to other non-related strategy tests...

Worth doing?

Reply:
That sounds like a reasonable thing to do before they start getting large

Reply:
Pushed in a separate commit.  Tell me if you hate it.